### PR TITLE
Make code comply with Pylint 1.7.0

### PR DESCRIPTION
**Pull request by Ichimonji10**
See: https://docs.pylint.org/en/latest/whatsnew/1.7.html

### CODE REVIEW

**Comment by Ichimonji10** [comment]
I'm marking this as DO NOT MERGE because the changes need to be tested against some real, live Pulp systems. Having this PR open still helps me tracks the work, though, and gives teammates a chance to see what's coming down the pike.

File: pulp_smash/tests/rpm/api_v2/test_rsync_distributor.py [hunk 1]
```diff
@@ -404,7 +403,7 @@ def test_predistributor_id(self):
                 repo = api_client.post(REPOSITORY_PATH, body)
         except AssertionError as err:
             self.addCleanup(api_client.delete, repo['_href'])
-            raise err
+            raise err  # pylint:disable=raising-bad-type
```
Comment (elyezer) [inline pulp_smash/tests/rpm/api_v2/test_rsync_distributor.py]: I wonder if just calling raise would allow removing the pylint:disable

Comment (Ichimonji10) [inline pulp_smash/tests/rpm/api_v2/test_rsync_distributor.py]: Great suggestion! This works perfectly.

**Comment by Ichimonji10** [comment]
I've executed all of the modified test cases against VMs, and they pass. I'm reasonably confident in these changes.

**Pull request merged**